ios/RNMobileDeviceManager.m

```diff
@@ -42,6 +42,34 @@ - (void) settingsDidChange:(NSDictionary<NSString *, id> *) changes {
                                                 body:appConfig];
 }
 
++ (void) isASAMSupported:(void(^)(BOOL))callback {
+    if (UIAccessibilityIsGuidedAccessEnabled()) {
+        UIAccessibilityRequestGuidedAccessSession(NO, ^(BOOL didSucceed) {
+            if (didSucceed) {
+                UIAccessibilityRequestGuidedAccessSession(YES, ^(BOOL didSuceed) {
+                    if (didSucceed)  {
+                        callback(YES);
```

Comment:
If it didn't succeed, it doesn't look like we're letting the invoker know.

This is where just doing `callback(didSucceed)` is handy because we have less edges to worry about